src/navigation/sectionalTabs/sectionalTabs.jsx

```diff
@@ -14,14 +14,17 @@ import {
 } from 'lodash';
 import PropTypes from 'prop-types';
 import ResizeDetector from 'react-resize-detector';
-import SectionalTab from './sectionalTab';
-import SectionalTabContent from './sectionalTabContent';
-import withStyles from '../../styles/withStyles';
-import DropdownButton from '../../inputs/dropdownButton';
 import {
     BEM_NAVIGATION_SECTIONAL_TABS,
     BEM_NAVIGATION_TAB_ROOT_CLASS,
 } from '../../global/constants';
+import {
+    PADDING_X,
+} from './constants';
+import SectionalTab from './sectionalTab';
+import SectionalTabContent from './sectionalTabContent';
+import withStyles from '../../styles/withStyles';
+import DropdownButton from '../../inputs/dropdownButton';
```

Comment:
NAB: Should this be sorted above Line 24 `import SectionalTab from './sectionalTab';` for some alphabetical ordering?

Reply:
Fixed.